dotnet/src/Connectors/Connectors.AI.OpenAI/AzureSdk/ChatStreamingResultExtensions.cs

```diff
@@ -29,9 +29,17 @@ public static class ChatStreamingResultExtensions
         FunctionCall? functionCall = null;
         await foreach (AzureOpenAIChatMessage message in chatStreamingResult.GetStreamingChatMessageAsync())
         {
-            functionCall ??= message.FunctionCall;
+            functionCall ??= message.InnerChatMessage?.FunctionCall;
 
-            arguments.Append(message.FunctionCall.Arguments);
+            if (functionCall is null && message.InnerChatMessage?.Name is not null)
```

Comment:
For some streaming scenarios, the `FunctionCall` is null but the function name is present in the `Name` property of the message.

Reply:
Reverted.